### DISCUSSION TITLE
Question about providing different tags to retried ops

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17780] [Q&A] [answered]
Hey, a quick question about op retries - is it possible to provide different tags to the retried op (post-failure)? In my case, I’d like to edit `dagster-k8s/config` and allocate more pod resources in case of `OOMKilled` event. Thanks

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699386632384489?thread_ts=1699386632.384489&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment, marked as answer]
I don't believe this is supported right now. I'll open an issue for it here https://github.com/dagster-io/dagster/issues/17783
